data/json/overmap/overmap_biomes.json

```diff
@@ -0,0 +1,31 @@
+[
```

Comment:
Pleas use [web-linter](http://dev.narc.ro/cataclysm/format.html) to format all json files you create or modify.

Reply:
Yeah, I will. I probably should do that sooner rather than later.